docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -2928,35 +2928,29 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "Name": "bridge",
-  "Id": "f2de39df4171b0dc801e8002d1d999b77256983dfc63041c0f34030aa3977566",
+  "Name": "net01",
+  "Id": "7d86d31b1478e7cca9ebed7e73aa0fdeec46c5ca29497431d3007d2d9e15ed99",
   "Scope": "local",
   "Driver": "bridge",
   "IPAM": {
     "Driver": "default",
     "Config": [
       {
-        "Subnet": "172.17.0.0/16"
+        "Subnet": "172.19.0.0/16",
+	"Gateway": "172.19.0.1/16"
       }
     ]
   },
   "Containers": {
-    "39b69226f9d79f5634485fb236a23b2fe4e96a0a94128390a7fbbcc167065867": {
-      "Name": "mad_mclean",
-      "EndpointID": "ed2419a97c1d9954d05b46e462e7002ea552f216e9b136b80a7db8d98b442eda",
-      "MacAddress": "02:42:ac:11:00:02",
-      "IPv4Address": "172.17.0.2/16",
+    "19a4d5d687db25203351ed79d478946f861258f018fe384f229f2efa4b23513c": {
+      "Name": "test",
+      "EndpointID": "628cadb8bcb92de107b2a1e516cbffe463e321f548feb37697cce00ad694f21a",
+      "MacAddress": "02:42:ac:13:00:02",
+      "IPv4Address": "172.19.0.2/16",
       "IPv6Address": ""
     }
   },
-  "Options": {
-    "com.docker.network.bridge.default_bridge": "true",
-    "com.docker.network.bridge.enable_icc": "true",
-    "com.docker.network.bridge.enable_ip_masquerade": "true",
-    "com.docker.network.bridge.host_binding_ipv4": "0.0.0.0",
-    "com.docker.network.bridge.name": "docker0",
-    "com.docker.network.driver.mtu": "1500"
-  }
```

Comment:
Can you add this back (with sensible values for the network?) I'd like to have something visible here, so that readers get an understanding what is in there :smile: